client/components/lessons/LessonNavigation/LessonNavigation.module.scss

```diff
@@ -0,0 +1,35 @@
+@import "../../../styles/variables";
+
+.lesson-navigation {
```

Comment:
Let's rename the same as the component `.LessonNavigation`.

I know initially this was the naming, and a lot of components still use it. But I think we should move away from this, and towards naming the same as the component. This way, when you search in the editor, you find everything related to it: the component itself, the styles, etc.

Reply:
Done